plugins/pom.xml

```diff
@@ -31,5 +31,32 @@
          <artifactId>jboss-logging</artifactId>
          <version>3.3.0.Final</version>
       </dependency>
+
+      <!-- needed to avoid receive NoClassDefFoundError: com/arjuna/ats/arjuna/common/arjPropertyManager
+           during build -->
+      <dependency>
+         <groupId>org.jboss.narayana.jta</groupId>
+         <artifactId>narayana-jta</artifactId>
+         <version>5.10.4.Final</version>
+         <optional>true</optional>
+         <exclusions>
+            <exclusion>
+               <groupId>org.jboss.logging</groupId>
+               <artifactId>jboss-logging-spi</artifactId>
+            </exclusion>
+         </exclusions>
+      </dependency>
+
    </dependencies>
+
+   <dependencyManagement>
+      <dependencies>
+         <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>2.6</version>
```

Comment:
Same here. This version should be in dev.properties or retrieved from here: https://github.com/infinispan/infinispan/blob/7de021c034851e56efaad8abc253422543a8b8cf/pom.xml#L1117